Userland/Applications/Mail/InboxModel.h

```diff
@@ -39,7 +44,8 @@ class InboxModel final : public GUI::Model {
 
     virtual ~InboxModel() override = default;
 
-    void set_seen(int row, bool);
+    MailStatus mail_status(int row);
```

Comment:
(can be const too)

Reply:
I did mark them as const originally but removed it because of the compiler error below:

`error: type qualifiers ignored on function return type [-Werror=ignored-qualifiers]`

Reply:
No, I mean the implicit this pointer can be const, like `MailStatus mail_status(int row) const;` (also on the definition)

Reply:
Ah, I see. Sorry for the misunderstanding!